frontend/src/lib/Container.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
```

Comment:
Nice that you have made a separate library for the styling and that you have them in different files. 💯 
Please check your indentations, you miss them for some of the styling components.